mafia/models.py

```diff
@@ -191,32 +192,32 @@ def vote(self, nodes):
             victim_name, _ = max(sorted_kv, key=lambda kv: kv[1])
             self.last_victim_name = victim_name
             victim_node = Node.query.filter_by(property1=victim_name).one()
-            victim_node.alive = 'False'
-            for v in victim_node.vectors():
-                v.fail()
-            for i in victim_node.infos():
-                i.fail()
-            for t in victim_node.transmissions(direction="all"):
-                t.fail()
-            for t in victim_node.transformations():
-                t.fail()
-            victim_node.deathtime = timenow()
+            self.kill_victim(victim_node)
         else:
             victim_name = None
         return victim_name
 
+    def kill_victim(self, victim_node):
+        """Don't fail the Node itself, but fail all it's related objects.
+        TODO: explain why not just victim_node.fail()
```

Comment:
I didn't find the reason for not failing the Node clear. A comment would be helpful.